src/index.js

```diff
@@ -138,10 +138,15 @@ class Dropzone extends React.Component {
     this.isFileDialogActive = false;
 
     fileList.forEach((file) => {
-      if (this.fileAccepted(file) && this.fileMatchSize(file)) {
-        if (!disablePreview) {
+      if (!disablePreview) {
```

Comment:
Makes me think if we should tackle the bigger problem in this PR: what if I want to `disablePreviews` based on file property. I was thinking about a callback that can be defined by the user that should return `true` or `false`. This way we could also use it to check if the method exists for this `file` instance and call it.

The problem here I think is that we mutate `file` by adding a new property to the object. But I'm wondering if this is an issue in this context.

Thoughts?